source/releasenotes/compat.rst

```diff
@@ -26,7 +26,7 @@ indicate compatibility with CentOS 6.2, 6.1 and so on.
 
 -  RHEL versions 6.3, 6.5, 6.6 and 7.0
 -  CentOS versions 6.8, 7
--  Ubuntu 14.04, 16.04, 17.04 LTS
+-  Ubuntu 14.04, 16.04, 18.04 LTS
```

Comment:
Didn't we drop support for 14.04? cc @wido @GabrielBrascher ? Or was it in 4.13/current-master?

Reply:
You are right @rhtyd, thanks for pointing that! I have updated the PR.

Reply:
Update the changes list, adding 'Drop support for Ubuntu 14.04'